supplementary_style_guide/glossary_terms_conventions/general_conventions/m.adoc

```diff
@@ -518,6 +518,40 @@ Azure CLI 2.0 is the most current command-line interface and is replacing Xplat-
 
 *See also*:
 
+[discrete]
+[[module]]
+==== image:images/yes.png[yes] module (noun)
+*Description*: In Red Hat Enterprise Linux, a _module_ is a collection of packages representing a logical unit: an application, a language stack, a database, or a set of tools. These packages are built, tested, and released together.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[module-stream]]
+==== image:images/yes.png[yes] module stream (noun)
+*Description*: In Red Hat Enterprise Linux, _module streams_ are filters that can be imagined as virtual repositories in the AppStream physical repository. Module streams represent versions of the Application Stream components.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[module-profile]]
+==== image:images/yes.png[yes] module profile (noun)
+*Description*: In Red Hat Enterprise Linux, a _module profile_ is a list of recommended packages to be installed together for a particular use case such as for a server, client, development, minimal install, or other. These package lists can contain packages outside the module stream, usually from the BaseOS repository or the dependencies of the stream.
```

Comment:
```suggestion
*Description*: In Red Hat Enterprise Linux, a _module profile_ is a list of recommended packages to be installed together for a particular use case such as for a server, client, development, minimal install, or other purpose. These package lists can contain packages outside the module stream, usually from the BaseOS repository or the dependencies of the stream.
```
- `other` needs a reference. Other what? Not sure if `purpose` is what you intend, `other` shouldn't be on its own here.

Reply:
+1

Reply:
Thank you, I changed it to "other use cases".